src/sentry/incidents/subscription_processor.py

```diff
@@ -247,7 +248,11 @@ def trigger_resolve_threshold(self, trigger):
             self.handle_incident_severity_update()
 
             if self.check_triggers_resolved():
-                update_incident_status(self.active_incident, IncidentStatus.CLOSED)
+                update_incident_status(
+                    self.active_incident,
+                    IncidentStatus.CLOSED,
+                    status_method=IncidentStatusMethod.SENTRY,
```

Comment:
Wondering if this one should be `ALERT_RULE` instead?

Reply:
Yeah, I like that better. I'll change it :-)

Reply:
Made it "RULE_TRIGGERED" to better offer contrast to "RULE_UPDATED"?